types/index.d.ts

```diff
@@ -11,6 +12,7 @@ declare global {
   interface Window {
     swarm: {
       web2Helper: Web2HelperContent
+      localStorage: LocalStorage
     }
     swarmSessionId: string
   }
```

Comment:
Wouldn't it be nicer to have `swarm.sessionId` instead of both `swarm` and `swarmSessionId` in the global namespace?

Not related to this change, but I'd suggest opening an issue about this for later. 🙂